lib/exercism/tooling_job.rb

```diff
@@ -91,7 +90,8 @@ def processed!
       redis = Exercism.redis_tooling_client
       redis.multi do
         redis.lrem(key_for_executed, 1, id)
-        redis.rpush(key_for_processed, id)
+        redis.del("job:#{id}")
```

Comment:
A bit of duplication here, but it's fine with the test you added to verify the behavior.